diem-move/diem-framework/experimental/sources/NFT.move

```diff
@@ -48,8 +48,18 @@ address 0x1 {
             amount: u64,
         }
 
+        struct TransferEvent has copy, drop, store {
+            id: GUID::ID,
+            from: address,
+            to: address,
```

Comment:
I'm not really a fan of `from` and `to`, but it looks like that's what we're already using.  I'd prefer sender and receiver, but you can leave it like this for now.